[debug] [2025-10-08T15:01:56.302Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-10-08T15:01:57.530Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-10-08T15:01:57.537Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-10-08T15:01:57.537Z] > authorizing via signed-in user (mjieduardo@gmail.com)
[debug] [2025-10-08T15:01:57.542Z] Checked if tokens are valid: false, expires at: 1759885273386
[debug] [2025-10-08T15:01:57.544Z] Checked if tokens are valid: false, expires at: 1759885273386
[debug] [2025-10-08T15:01:57.544Z] > refreshing access token with scopes: []
[debug] [2025-10-08T15:01:57.547Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-10-08T15:01:57.547Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-10-08T15:01:57.563Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.13.0) on port 50001
[debug] [2025-10-08T15:01:58.114Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E1008 08:01:58.107031    9336 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E1008 08:01:58.107031    9336 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:01:58.116Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-10-08T15:01:58.116Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-10-08T15:01:58.116Z] Starting Data Connect Emulator with command {"binary":"C:\\Users\\Lenovo\\.cache\\firebase\\emulators\\dataconnect-emulator-2.13.0.exe","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"C:\\\\Users\\\\Lenovo\\\\.cache\\\\firebase\\\\emulators\\\\dataconnect-emulator-2.13.0.exe\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to \u001b[1mdataconnect-debug.log\u001b[22m"}}
[debug] [2025-10-08T15:01:58.234Z] E1008 08:01:58.158505   18040 dev.go:79] Failed to load initial config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:01:58.161274   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
I1008 08:01:58.180011   18040 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:01:58.158505   18040 dev.go:79] Failed to load initial config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:01:58.161274   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\nI1008 08:01:58.180011   18040 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-10-08T15:01:58.264Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-10-08T15:01:58.265Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-10-08T15:01:58.300Z] [Firebase Plugin] User found:  mjieduardo@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-10-08T15:01:58.302Z] Checked if tokens are valid: true, expires at: 1759939317265
[debug] [2025-10-08T15:01:58.303Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[error] [Firebase Plugin] error:  {}
[debug] [2025-10-08T15:01:58.634Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-10-08T15:01:58.635Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-10-08T15:02:08.165Z] E1008 08:02:08.162653   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:08.162653   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:02:18.163Z] E1008 08:02:18.162880   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:18.162880   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:02:28.162Z] E1008 08:02:28.162671   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:28.162671   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:02:38.162Z] E1008 08:02:38.162907   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:38.162907   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:02:48.163Z] E1008 08:02:48.163322   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:48.163322   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:02:58.163Z] E1008 08:02:58.163001   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:02:58.163001   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:08.163Z] E1008 08:03:08.163093   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:08.163093   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:18.162Z] E1008 08:03:18.162944   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:18.162944   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:28.163Z] E1008 08:03:28.162952   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:28.162952   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:38.163Z] E1008 08:03:38.163203   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:38.163203   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:48.163Z] E1008 08:03:48.163157   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:48.163157   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:58.165Z] E1008 08:03:58.163773   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:58.163773   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:03:58.167Z] E1008 08:03:58.165335   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:03:58.165335   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:04:08.163Z] E1008 08:04:08.163196   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:08.163196   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:04:18.164Z] E1008 08:04:18.163544   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:18.163544   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:04:28.163Z] E1008 08:04:28.162847   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:28.162847   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:04:38.164Z] E1008 08:04:38.164068   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:38.164068   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:04:48.163Z] E1008 08:04:48.163533   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:48.163533   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:04:58.162Z] E1008 08:04:58.162689   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:04:58.162689   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:08.159Z] E1008 08:05:08.162878   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:08.162878   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:18.160Z] E1008 08:05:18.162888   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:18.162888   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:28.159Z] E1008 08:05:28.163175   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:28.163175   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:38.160Z] E1008 08:05:38.163968   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:38.163968   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:48.160Z] E1008 08:05:48.163618   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:48.163618   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:05:58.160Z] E1008 08:05:58.163001   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:05:58.163858   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:05:58.163001   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:05:58.163858   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:06:08.164Z] E1008 08:06:08.164606   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:08.164606   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:06:18.163Z] E1008 08:06:18.163117   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:18.163117   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:06:28.162Z] E1008 08:06:28.162992   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:28.162992   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:06:38.163Z] E1008 08:06:38.162851   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:38.162851   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:06:48.162Z] E1008 08:06:48.162150   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:48.162150   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:06:58.162Z] E1008 08:06:58.162644   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:06:58.162644   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:08.162Z] E1008 08:07:08.162884   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:08.162884   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:18.163Z] E1008 08:07:18.162702   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:18.162702   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:28.162Z] E1008 08:07:28.162822   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:28.162822   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:38.162Z] E1008 08:07:38.162870   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:38.162870   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:48.163Z] E1008 08:07:48.162299   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:48.162299   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:07:58.163Z] E1008 08:07:58.162987   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:07:58.162987   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:07:58.162987   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:07:58.162987   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:08:08.162Z] E1008 08:08:08.162826   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:08.162826   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:08:18.164Z] E1008 08:08:18.163456   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:18.163456   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:08:28.163Z] E1008 08:08:28.162549   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:28.162549   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:08:38.163Z] E1008 08:08:38.163679   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:38.163679   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:08:48.165Z] E1008 08:08:48.164726   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:48.164726   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:08:58.164Z] E1008 08:08:58.163176   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:08:58.163176   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:08.164Z] E1008 08:09:08.163816   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:08.163816   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:18.164Z] E1008 08:09:18.163797   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:18.163797   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:28.164Z] E1008 08:09:28.163209   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:28.163209   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:38.164Z] E1008 08:09:38.163925   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:38.163925   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:48.162Z] E1008 08:09:48.162395   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:48.162395   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:09:58.163Z] E1008 08:09:58.162704   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:09:58.163509   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:09:58.162704   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:09:58.163509   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:10:08.163Z] E1008 08:10:08.163393   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:08.163393   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:10:18.163Z] E1008 08:10:18.163237   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:18.163237   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:10:28.161Z] E1008 08:10:28.162755   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:28.162755   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:10:38.161Z] E1008 08:10:38.162650   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:38.162650   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:10:48.161Z] E1008 08:10:48.162611   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:48.162611   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:10:58.161Z] E1008 08:10:58.162549   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:10:58.162549   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:08.161Z] E1008 08:11:08.162646   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:08.162646   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:18.161Z] E1008 08:11:18.163239   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:18.163239   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:28.162Z] E1008 08:11:28.162973   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:28.162973   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:38.163Z] E1008 08:11:38.162723   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:38.162723   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:48.163Z] E1008 08:11:48.162857   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:48.162857   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:11:58.163Z] E1008 08:11:58.162930   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:11:58.162930   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:11:58.162930   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:11:58.162930   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:12:08.162Z] E1008 08:12:08.162225   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:08.162225   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:12:18.163Z] E1008 08:12:18.162630   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:18.162630   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:12:28.163Z] E1008 08:12:28.163111   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:28.163111   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:12:38.164Z] E1008 08:12:38.163924   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:38.163924   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:12:48.163Z] E1008 08:12:48.162890   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:48.162890   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:12:58.162Z] E1008 08:12:58.162505   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:12:58.162505   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:08.162Z] E1008 08:13:08.162776   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:08.162776   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:18.162Z] E1008 08:13:18.162636   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:18.162636   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:28.162Z] E1008 08:13:28.162250   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:28.162250   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:38.162Z] E1008 08:13:38.163201   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:38.163201   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:48.162Z] E1008 08:13:48.162785   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:48.162785   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:13:58.163Z] E1008 08:13:58.164243   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
E1008 08:13:58.164243   18040 watch.go:102] watchAllDir unable to walk dir `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`: CreateFile c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect: The system cannot find the file specified.
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:13:58.164243   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\nE1008 08:13:58.164243   18040 watch.go:102] watchAllDir unable to walk dir `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`: CreateFile c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect: The system cannot find the file specified.\n"}}
[debug] [2025-10-08T15:14:08.161Z] E1008 08:14:08.162121   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:14:08.162121   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:14:18.162Z] E1008 08:14:18.163065   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:14:18.163065   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:14:28.161Z] E1008 08:14:28.161978   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:14:28.161978   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:14:38.162Z] E1008 08:14:38.162849   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:14:38.162849   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
[debug] [2025-10-08T15:14:48.162Z] E1008 08:14:48.163199   18040 watch.go:31] Failed to load config from "c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect": could not find dataconnect.yaml in `c:\Users\Lenovo\Desktop\Quasar\QuasarPercoGPS\dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E1008 08:14:48.163199   18040 watch.go:31] Failed to load config from \"c:\\\\Users\\\\Lenovo\\\\Desktop\\\\Quasar\\\\QuasarPercoGPS\\\\dataconnect\": could not find dataconnect.yaml in `c:\\Users\\Lenovo\\Desktop\\Quasar\\QuasarPercoGPS\\dataconnect`\n"}}
